lib/index.ts

```diff
@@ -28,7 +28,8 @@ export * from "./osml/model_endpoint/me_http_endpoint";
 export * from "./osml/osml_table";
 export * from "./osml/osml_topic";
 export * from "./osml/osml_vpc";
-export * from "./osml/osml_authenticate";
+export * from "./osml/osml_auth";
+export * from "./osml/osml_auth_server";
```

Comment:
the file should probably be renamed to osml_auth_alb to match

Reply:
ty ty

Reply:
fixed